docs/index.rst

```diff
@@ -152,6 +159,21 @@ There are some additional tools, mostly of use for maintainers, in the
 `astropy/astropy-procedures repository
 <https://github.com/astropy/astropy-procedures>`__.
 
+{%else%}
+
+The developer documentation contains instructions for how to contribute to
+Astropy or affiliated packages, as well as coding, documentation, and
```

Comment:
Since we are removing the "install the developer version" above, how about adding here:
"...affiliated pacakges, install and test the development version, as well as ..."